### ISSUE TITLE
Better Extension Name

### ISSUE THREAD

**Issue by akaroml** [redhat-developer/vscode-java#807]
Congratulations!
https://marketplace.visualstudio.com/VSCode

<img width="1267" alt="screen shot 2019-02-22 at 8 46 48 am" src="https://user-images.githubusercontent.com/16755094/53212113-9e3ba900-367e-11e9-8e69-c783b8c1049a.png">

vscode-java made its way to the most popular extension list and now on the frontpage of vscode marketplace. This is really a nice thing to gain more exposure and reaching out to more users.

But by only looking at the item itself, it's a bit hard to tell this is a Java extension. Users only see "Language Server for...", only without Java.

I would suggest that we optimize the name for better exposure by putting Java first. Something like "Java Language Server by Red Hat TM".

**Comment by superadmini** [comment]
I very much agree！

**Comment by fbricon** [comment]
I agree, but this is in the hands of the Red Hat legal department. The Java trademark has some strict restrictions, we can't just do whatever we want.

**Comment by brunoborges** [comment]
Hey @fbricon any update on this? "Java Language Support" should check all legal boxes.
